neo/Cryptography/ECC/ECFieldElement.cs

```diff
@@ -112,30 +113,32 @@ public ECFieldElement Sqrt()
             BigInteger Q = this.Value;
             BigInteger fourQ = (Q << 2).Mod(curve.Q);
             BigInteger U, V;
-            do
+            using (RandomNumberGenerator rng = RandomNumberGenerator.Create())
```

Comment:
Why use `RandomNumberGenerator` in `Sqrt()`?

Reply:
It's recommended to use a more secure and standardized function to replace `Random` here.

Reply:
I don't know how `RandomNumberGenerator` improves security in `Sqrt()`.

Reply:
You're right, it dosen't improve security here. I'll reset it and consider removing `ECDsa` in Neo2.x, as @vang1ong7ang said.